components/dashboard/src/admin/UserDetail.tsx

```diff
@@ -169,6 +173,12 @@ export default function UserDetail(p: { user: User }) {
                             }] :  undefined}
                         >{isStudent === undefined ? '---' : (isStudent ? 'Enabled' : 'Disabled')}</Property>
                     </div>
+                    <div className="flex w-full mt-6 overflow-scroll">
+                        {authProviders && authProviders.length > 0 &&
+                            <Property name="Git providers">
+                                <div className="overflow-scroll">{authProviders?.join(", ")}</div>
```

Comment:
**suggestion**: Don't want to block this but what do you think of something like this? Happy to discuss this in a separate follow up issue.

| Collapsed | Expanded |
|-|-|
| <img width="695" alt="providers-collapsed" src="https://user-images.githubusercontent.com/120486/157537513-1c167f21-a3a6-4303-99e0-36aae038d561.png"> | <img width="695" alt="providers-expadned" src="https://user-images.githubusercontent.com/120486/157537397-eae55ff7-bd77-4486-916e-6eb5474bb3ec.png"> |

Reply:
Removing from [in progress](https://github.com/orgs/gitpod-io/projects/29), let me know if there's any more design feedback needed.

Reply:
@gtsiolis Sorry I dropped the ball here - will ping you once I pick it back up!